doc/index.rst

```diff
@@ -44,6 +44,15 @@
 
     discussion/*
 
+.. toctree::
```

Comment:
I think tutorials should be before the how-tos. i.e. the first category.

Reply:
f9c97cb5